graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpTransport.java

```diff
@@ -294,6 +297,15 @@ public ConfigurationRequest getRequestedConfiguration() {
                     )
             );
 
+            cr.addField(
+                    new BooleanField(
+                            CK_TLS,
+                            "Enable TLS?",
+                            false,
+                            "Enable transport encryption via TLS. (NEEDS correct TLS port setting!)"
```

Comment:
The all-caps word in the parentheses looks strange. Maybe rephrase to "Enable transport encryption via TLS (requires valid TLS port setting."